src/RichMentionsProvider.tsx

```diff
@@ -295,10 +295,31 @@ export function RichMentionsProvider<T = object>({
     config: TMentionConfig<T>
   ): void {
     const fixed = ref.current.fixed;
-    const rect = node.getBoundingClientRect();
-    const y = fixed ? 0 : window.pageYOffset;
-    const x = fixed ? 0 : window.pageXOffset;
+    const rect = { top: 0, right: 0, bottom: 0, left: 0 };
```

Comment:
What is the difference of this rect with the previous code ?
Clearer ?

Reply:
The type returned by typescript is readonly so I have to write my own object to update its fields.